### DISCUSSION TITLE
[Prettier Plugin] Migrate to Prettier v3

### DISCUSSION THREAD

**Discussion by sosukesuzuki** [tailwindlabs/tailwindcss#9602] [Ideas]
Hi, I am a maintainer at Prettier.

The Prettier team is currently working on v3(still alpha).
v3 will include breaking changes to the plugin interface.
Please see https://github.com/prettier/prettier/releases/tag/3.0.0-alpha.2 for more information.

I created this issue to help major plugins migrate to Prettier v3.

Myself and @fisker have created a migration guide which you can see here:

https://github.com/prettier/prettier/wiki/How-to-migrate-my-plugin-to-support-Prettier-v3%3F

If you have any questions, feel free to mention us. We can help you.

If you have already migrated to Prettier v3, sorry but please close this issue.

**Comment by DrJume** [comment]
https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/176#issuecomment-1630559854
